src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmRemoveApplication.cshtml

```diff
@@ -1,6 +1,5 @@
-@* TODO: This is a Holding Page. Please use appropriate viewmodel when implementing *@
 @using SFA.DAS.AdminService.Common.Extensions;
-@model SFA.DAS.RoatpGateway.Web.ViewModels.RoatpGatewayOutcomeViewModel
```

Comment:
I wouls suggest leave this as RoatpGatewayOutcomeViewModel, will make future reuse of partials easier? Your choice tho